hack/test-extended/ldap/run.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# This scripts starts the OpenShift server with a default configuration.
+# The OpenShift Docker registry and router are installed.
+# It will start all 'default_*_test.go' test cases.
+
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
+cd "${OS_ROOT}"
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+
+cleanup() {
+	out=$?
+	set +e
+	
+	echo "[INFO] Tearing down test"
+	kill_all_processes
+	rm -rf ${ETCD_DIR-}
+	echo "[INFO] Stopping k8s docker containers"; docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
```

Comment:
This is probably common to a lot of scripts, include the flag name right?  If so, move to `util.sh`